src/common/constants.ts

```diff
@@ -1,4 +1,6 @@
 const MAP_API_KEY = process.env.GATSBY_MAP_API_KEY
+const GATSBY_GROWTHBOOK_CLIENT_KEY =
+    process.env.GATSBY_GROWTHBOOK_CLIENT_KEY || 'sdk-0aHHaxKhwn0bP1HE'
```

Comment:
```suggestion
    process.env.GATSBY_GROWTHBOOK_CLIENT_KEY'
```

Reply:
done, also add in vercel to build successfully